example.env

```diff
@@ -2,8 +2,8 @@ PORT=5000
 PGUSER='ubuntu'
 PGPORT='5432'
 PGDATABASE='waggl'
-PGHOST='insertEC2here'
-PGPASSWORD='weshouldcomeupwithone'
+PGHOST='34.229.167.164'
+PGPASSWORD='password'
```

Comment:
might not want to hardcode this into the example file if this is the actual password ...?

Reply:
i think we should share these configs in slack for security purposes!

Reply:
It's DEFINITELY not the real password ;), but the example.env is gone now, you're completely right.